frontend/src/metabase/admin/datamodel/containers/SegmentApp.unit.spec.tsx

```diff
@@ -33,7 +33,11 @@ interface SetupOpts {
 const setup = ({ initialRoute = FORM_URL }: SetupOpts = {}) => {
   setupDatabasesEndpoints([createSampleDatabase()]);
   setupSearchEndpoints([]);
-  setupCardDataset();
+  setupCardDataset({
+    data: {
+      rows: [[null]],
```

Comment:
By default `rows` is an empty array but `updatePreviewSummary` in `datamodel.js` expects at least 1 row.